controllers/networking-calico/example/30-crd-managed-resources.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
group all crd examples together - rename to `10-crd-...`

Reply:
This comment is not yet resolved. I still see files `10-crd-network` and `20-crd-cluster`. Group them together for a better overview, please.

Reply:
network is different than cluster. I grouped the CRD and the Network resource under 10. The Cluster resource was different. Maybe I mis-understood how it is used, but to me lexiographical order meant that each type gets a number.

Reply:
Take a look at how it's done [here](https://github.com/gardener/gardener-extensions/tree/master/controllers/provider-aws/example).